pkg/discovery/processor/controller_processor.go

```diff
@@ -97,7 +97,7 @@ func (cp *ControllerProcessor) cacheAllControllers() {
 			}
 			continue
 		}
-		for _, item := range list.Items {
+		for _, item := range items {
```

Comment:
Is it possible to only cache the active controllers? For example:

* Skip `ReplicaSet` that has a `Deployment` owner, but has 0 replicas
* Similar logic to skip inactive `ReplicationController` that has `DeploymentConfig` owner

Reply:
@ading1977  Updated the code to implement this, PTAL!